server/config.coffee

```diff
@@ -33,7 +33,8 @@ config =
             americano.errorHandler
                 dumpExceptions: true
                 showStack: true
-            americano.static path.join __dirname, '/../client/public'
+            americano.static path.join __dirname, '../static'
```

Comment:
Please don't change too much things at a time. We have a clear convention for client and server folders in all our folders. It's going to be a total mess, there are so already so many differences between projects.